doc/README.md

```diff
@@ -23,9 +23,94 @@ need to provide both those things. The
 project](#generating-a-binary-installer-for-your-dune-project) section
 provides a good example on how to build them.
 
+Given that the information in the `oui.json` file is closely tighted to your
+project, we recommend committing it to the root of your repo.
+
 ## `oui.json` file format
 
-*TODO*
+The `oui.json` config file is composed of a main JSON object. The fields
+are:
+
+- `name`, **string**, **required**: the name of the app. This will define the
+  install folder name on the target system and where plugins will look for your
+  app when installing themselves. *Example:* `"oui"`.
+- `fullname`, **string**, **required**: *TODO* (remove or make wix specific?)
+- `version`, **string**, **required**: the version of the app. *Example:*
+  `"1.0.0"`.
+- `exec_files`, **string array**, **required**: The list of executables to
+  install from the bundle. Should be a list of paths, relative to the bundle
+  root, pointing to executable files that should be installed and made available
+  to the user. *Example:* `["bin/oui", "bin/opam-oui"]`.
+- `manpages`, **object**, **optional**: A JSON object describing where manpages
+  are located within the bundle so they can be properly installed on the target
+  system. See the [manpages object section](#manpages-object) for the object
+  format.
+- `environement`, **string array array**, **optional**: A list of environment
+  variables and associated values to set/unset in Windows terminal on
+  install/uninstall respectively. *Example:*
+  `[["VAR1", "value1"], ["VAR2", "value2"]]`.
+- `unique_id`, **string**, **required**: A unique identifier for the app in
```

Comment:
Might be worth mentionning that it corresponds to the bundle ID on MacOS and the package Unique ID on Windows.
Also note the field is required, so how do we deal with it on Linux ? Or do we also make it optional ?